cesm/flux_atmocn/shr_flux_mod.F90

```diff
@@ -257,22 +263,21 @@ SUBROUTINE flux_atmOcn(logunit, nMax  ,zbot  ,ubot  ,vbot  ,thbot ,   &
     real(R8)    :: tdiff(nMax)               ! tbot - ts
     real(R8)    :: vscl
 
+    real(R8)    :: ugust      ! function: gustiness as a function of convective rainfall.  
+    real(R8)    :: gprec   ! dummy arg ~ ?
```

Comment:
Not a great comment here.

Reply:
Updated this too; working on the esmf info object now.